gtn/functions/compose.cpp

```diff
@@ -52,14 +52,22 @@ void epsilonReachable(
   }
 }
 
-/* Find any state in the new composed graph which can reach
- * an accepting state. */
+/*
+ * Find any state in the new composed graph which can reach
```

Comment:
Not sure if I understand correctly, you are searching the graph backwards?  Don't know if this is part of a symmetric operation where the forwards is also done.  In speech recognition we tend to use graphs that are optimized for the forward direction of search, where the backwards one might be quite slow (e.g. "L"). 

A note on notation: in the literature, "accessible" means accessible from start state, and "coaccessible" means it can reach the end state.  Using a single word "reachable" to mean "coaccessible" might be a little confusing.

Reply:
The graph is essentially being pruned from the backward direction prior to the compose operation, and it's a fair point this might be suboptimal for many speech applications.

And yes co-accessible is the term we should use here to be in line with the lit.